### DISCUSSION TITLE
How many squares of any size do not contain a rook? ♟️

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1196] [Galaxy Trivia] [answered]
![image](https://github.com/ParanoidUser/yolo/assets/5120290/ab00485d-3e12-4b55-9b1b-4def9b2896b7)

**Comment by peterbaccaro** [comment, marked as answer]
There are **110 squares** without a rook.

There are 60 squares of size 1×1.
There are 35 squares of size 2×2.
There are 12 squares of size 3×3.
There are 3 squares of size 4×4.

A total of 60 + 35 + 12 + 3 = 110.

**Comment by Sajansharma0017** [comment]
110
